ext/storage_sources/s3_store/S3StoreLogSystem.h

```diff
@@ -0,0 +1,31 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+
+#include <aws/core/Core_EXPORTS.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+#include <aws/core/utils/logging/LogLevel.h>
+
+#include <atomic>
+
+class S3StoreLogSystem : public Aws::Utils::Logging::LogSystemInterface {
+
+    public:
+    explicit S3StoreLogSystem(WT_EXTENSION_API *wtApi, uint32_t awsVerbose);
+    ~S3StoreLogSystem() = default;
+    Aws::Utils::Logging::LogLevel
+    GetLogLevel(void) const override
+    {
+        return logLevel;
+    }
+    void Log(
+      Aws::Utils::Logging::LogLevel logLevel, const char *tag, const char *format, ...) override;
+    void LogStream(Aws::Utils::Logging::LogLevel logLevel, const char *tag,
+      const Aws::OStringStream &messageStream) override;
+    void Flush() override;
+
+    private:
+    void LogVerboseMessage(
```

Comment:
This function might be declared as const as it does not modify any member of the class:
`void LogVerboseMessage(...) const;`

Reply:
Changed! Thanks.